specification/keyvault/Security.KeyVault.Administration/client.tsp

```diff
@@ -16,10 +16,90 @@ namespace ClientCustomizations;
 @@clientName(KeyVault.SettingTypeEnum, "KeyVaultSettingType", "java");
 
 //C# customizations
+@client(
+    {
+      name: "KeyVaultAccessControlRestClient",
+      service: KeyVault,
+    },
+    "csharp"
+  )
+  @operationGroup
+    interface KeyVaultAccessControlRestClient{
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        deleteRoleDefinition is KeyVault.RoleDefinitions.delete;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        createOrUpdateRoleDefinition is KeyVault.RoleDefinitions.createOrUpdate;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        getRoleDefinition is KeyVault.RoleDefinitions.get;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        listRoleDefinitions is KeyVault.RoleDefinitions.list;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        deleteRoleAssignment is KeyVault.RoleAssignments.delete;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        createRoleAssignment is KeyVault.RoleAssignments.create;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        getRoleAssignment is KeyVault.RoleAssignments.get;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        listRoleAssignments is KeyVault.RoleAssignments.listForScope;
+    }
+
+    @client(
+        {
+        name: "KeyVaultRestClient",
+        service: KeyVault,
+        }, 
+        "csharp")
+      interface KeyVaultClient {
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        fullBackup is KeyVault.fullBackup;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        preFullBackup is KeyVault.preFullBackup;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        fullRestoreOperation is KeyVault.fullRestoreOperation;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        preFullRestoreOperation is KeyVault.preFullRestoreOperation;
+        #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+        fullBackupStatus is KeyVault.fullBackupStatus;
```

Comment:
Per the [validation errors](https://github.com/Azure/azure-rest-api-specs/actions/runs/13554153659/job/37884604235?pr=32864#step:4:61), you'll need to also add a suppression for the non-standard method naming for this and the other status methods:
```suggestion
        #suppress "@azure-tools/typespec-azure-core/use-standard-names" "Operation name is already established"
        fullBackupStatus is KeyVault.fullBackupStatus;
```

Reply:
Thanks @mccoyp